superset-frontend/src/dashboard/components/URLShortLinkButton/index.tsx

```diff
@@ -66,7 +75,14 @@ export default function URLShortLinkButton({
       trigger="click"
       placement={placement}
       content={
-        <div id="shorturl-popover" data-test="shorturl-popover">
+        // eslint-disable-next-line jsx-a11y/no-static-element-interactions
```

Comment:
hmm, shouldn't the stop propagation go on the copy to clipboard link instead? It's weird that we need to handle a click on a div